### ISSUE TITLE
log highlights

### ISSUE THREAD

**Issue by yegor256** [yegor256/rultor#567]
Would be a nice feature, if we parse the `stdout` file after the finish of a build, and fetch "highlights" from there. Highlight is a text line that starts with some special prefix and contains a Markdown after it. For example:

```
RULTOR: [version 0.5.6](http://search.maven.org) was released to Maven Central
```

We should find all such lines, and show them in a Github message, one by one in an unordered list.

<hr id='w'>
- `567-359f8c66`/#934 (by )

**Comment by alex-palevsky** [comment]
@yegor256 once `567-359f8c66`/#934 puzzle is resolved (later, in another ticket), this ticket will be fully complete

**Comment by alex-palevsky** [comment]
@yegor256 last puzzled solved `567-359f8c66`/#934
